src/logic/safe/store/actions/fetchSafe.ts

```diff
@@ -123,9 +123,13 @@ export const checkAndUpdateSafe = (safeAdd: string) => async (dispatch: Dispatch
       modules: buildModulesLinkedList(modules?.array, modules?.next),
       nonce: Number(remoteNonce),
       threshold: Number(remoteThreshold),
+      featuresEnabled: localSafe?.currentVersion
+        ? enabledFeatures(localSafe?.currentVersion)
+        : localSafe?.featuresEnabled,
     }),
   )
 
+  if (!remoteOwners) return
```

Comment:
should this be part of this PR?

Reply:
No, I just added that to avoid getting errors while develop, I think @matextrem add the best solution in his PR